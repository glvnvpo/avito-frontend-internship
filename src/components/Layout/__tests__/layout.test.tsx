import React from 'react';
import {mount} from 'enzyme';
import {Layout} from '../layout';

describe('Layout', () => {
	let wrapper;

	it('should render correct layout', () => {
		wrapper = mount(<Layout/>);
		expect(wrapper).toMatchSnapshot();
	});
});