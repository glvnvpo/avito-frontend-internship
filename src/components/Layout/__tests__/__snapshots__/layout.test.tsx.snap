// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render correct layout 1`] = `
<Layout>
  <PageHeader
    className="layout bg-orange"
    title="HACKER NEWS"
  >
    <ResizeObserver
      onResize={[Function]}
    >
      <SingleObserver
        key="rc-observer-key-0"
        onResize={[Function]}
      >
        <DomWrapper>
          <div
            className="ant-page-header layout bg-orange ant-page-header-ghost"
          >
            <div
              className="ant-page-header-heading"
            >
              <div
                className="ant-page-header-heading-left"
              >
                <span
                  className="ant-page-header-heading-title"
                  title="HACKER NEWS"
                >
                  HACKER NEWS
                </span>
              </div>
            </div>
          </div>
        </DomWrapper>
      </SingleObserver>
    </ResizeObserver>
  </PageHeader>
  <Outlet />
</Layout>
`;
